app/helpers/cloud_subnet_helper/textual_summary.rb

```diff
@@ -76,6 +76,21 @@ def textual_network_router
     @record.network_router
   end
 
+  def textual_parent_subnet
+    @record.parent_cloud_subnet
+  end
+
+  def textual_managed_subnets
+    label = _("Managed Subnets")
```

Comment:
I think this whole method can be just 

@record.cloud_subnets

Can you try it? It can be in the next PR